index.js

```diff
@@ -14,34 +24,82 @@ var allowCrossDomain = function (req, res, next) {
     }
 };
 
-function getPathFromUrl(url) {
+const getPathFromUrl = (url) => {
```

Comment:
I would extract that to separate module (like utils or so) and leave only the controllers in this file.